src/main/java/frc/robot/RobotContainer.java

```diff
@@ -124,6 +128,8 @@ private void configureBindings() {
     driverController.x().onTrue(enterXMode);
     driverController.a().onTrue(runIntake);
```

Comment:
remove the runIntake button mapping. We don't want A to run 2 very similar commands at the same time.